os-client/src/main/kotlin/io/provenance/os/client/OsClient.kt

```diff
@@ -173,7 +174,7 @@ open class OsClient(uri: URI) {
             dime.dime,
             dime.encryptedPayload,
             uuid = uuid,
-            metadata = metadata + (SIGNATURE_PUBLIC_KEY_FIELD_NAME to CertificateUtil.publicKeyToPem(signingKeyPair.public)),
+            metadata = metadata + (SIGNATURE_PUBLIC_KEY_FIELD_NAME to CertificateUtil.publicKeyToPem(signerPublicKey)),
```

Comment:
probably can replace the CertificateUtil.publicKeyToPem(signerPublicKey) with `signingPublicKey` from above (to avoid computing the same value twice)

Reply:
ill fix this up